src/imap_mag/client/sdcDataAccess.py

```diff
@@ -76,16 +73,21 @@ def download(self, file_name: str) -> pathlib.Path:
         pass
 
 
-class SDCApiClient(ISDCApiClient):
+class SDCDataAccess(ISDCDataAccess):
     """Class for uploading and downloading MAG data via imap-data-access."""
 
-    def __init__(self, data_dir: str) -> None:
+    def __init__(self, data_dir: str, sdc_url: str | None = None) -> None:
+        """Initialize SDC API client."""
+
         imap_data_access.config["DATA_DIR"] = pathlib.Path(data_dir)
+        imap_data_access.config["DATA_ACCESS_URL"] = (
+            sdc_url or "https://api.dev.imap-mission.com"
+        )
```

Comment:
Think I would be tempted to ditch the config object and the magic strings and just have properties on the `SDCDataAccess` object?

Reply:
Not sure I get it... The config is how the SDC lets us configure the URL (we can also use env variables, but this is consistent with the WebPODA approach).

Reply:
Oh I see sorry, think this was me not understanding